Aptoide/aptoide-client-v8
# [APP-648] Removed mopub ads

**What does this PR do?**

This PR aims at removing mopub ads. Removed dependencies, code and keys.

**Database changed?**

  No

**Where should the reviewer start?**

- [ ] MoPubAdsManager.java

**How should this be manually tested?**

Run the app and check that there are no ads. Also test installing the wallet and check that everything related with the super property is ok (now we dont make the call to wallet ads offer, we only check if the wallet is installed).
Also make sure everything is correct on home, search and appview.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-648](https://aptoide.atlassian.net/browse/APP-648)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
